### ISSUE TITLE
Windows mouse Mouse sensitivity not used for camera rotation

### ISSUE THREAD

**Issue by viper19861986** [MaFi-Games/Captain-of-Industry-issues#1626]
### Issue description

When using MMB = move mouse to rotate the camera around. I have noticed that it doesn't use the windows Mouse sensitivity speed, but rather direct raw DPI value. 
Believe it should also take into account the mouse sensitivity settings on the computer.

### Reproduction steps

Take note of mouse DPI and windows mouse sensitivity settings (800 DPI and 10 with windows mouse sensitivity)
Spin camera around in game
Change DPI settings to a very high or low value (3200 DPI)
Use windows mouse sensitivity setting to put back the mouse to the same realitive speed (changed to 2.5 (keeping in step with DPI change)
Mouse movement should "feels normal"
Pan around camera, it will be very fast or slow, depending on DPI setting. (in my case, mega fast)

### Game version and build number

v0.6.4d b343

### Operating system

Windows

**Comment by Appulcake** [comment]
I think one step better would be to just have a way to adjust camera sensitivity with a slider in game options, like many games provide. This way you can just fine tune the sensitivity of the free camera rotation/pitch to your likings, regardless of windows cursor speed, mouse DPI and so on.

Added a suggestion to that on their suggestions/ideas site: https://ideas.captain-of-industry.com/suggestions/655218/camera-sensitivity-speed-option

**Comment by jomfer** [comment]
Yes please! Just started the game and was confused that there was no camera speed option! Specifically the rotation (default middle mouse button) is way too fast by default for my liking, but I can't find any way to change it.

**Comment by viper19861986** [comment]
Well I think the mouse speed is tied to windows settings? It just is the MMB isn't.
But yeah in game setting for mouse speed and camera rotation via raw mouse input would be best!

**Comment by sprocketaudio** [comment]
Just bought this and game is unplayable even with low mouse dpi.
